packages/wicked/1007-include-mac-address-in-leaseinfo.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/src/leaseinfo.c b/src/leaseinfo.c
+index c47241f5..95f7fb0e 100644
+--- a/src/leaseinfo.c
++++ b/src/leaseinfo.c
+@@ -692,6 +692,18 @@ __ni_leaseinfo_dump(FILE *out, const ni_addrconf_lease_t *lease,
+ {
+ 	unsigned int i;
+ 	char *key = NULL;
++	ni_netconfig_t *nc = ni_global_state_handle(0);
++	ni_netdev_t *dev;
++
++	/* Find the network device using the name, and print the MAC in then
++	 * lease file */
++	if (!(dev = ni_netdev_by_name(nc, ifname))) {
++		ni_error("Unable to find interface with name %s", ifname);
++		return;
++	}
```

Comment:
Rather than erroring out, maybe emit a warning message and skip writing the `MACADDR` field.